ui/raidboss/browser_tts_engine.ts

```diff
@@ -30,6 +30,7 @@ export default class BrowserTTSEngine {
 
   constructor(private cactbotLang: Lang) {
     if (window.speechSynthesis !== undefined)
+	  window.speechSynthesis.getVoices();
       window.speechSynthesis.onvoiceschanged = () => this.initializeVoice();
     else
       console.error('BrowserTTS error: no browser support for window.speechSynthesis');
```

Comment:
```suggestion
    if (window.speechSynthesis !== undefined) {
      // https://bugs.chromium.org/p/chromium/issues/detail?id=334847
      window.speechSynthesis.onvoiceschanged = () => this.initializeVoice();
  	  window.speechSynthesis.getVoices();
    } else {
      console.error('BrowserTTS error: no browser support for window.speechSynthesis');
    }
```

style, but also I think you need to set the event before calling getVoices.

Reply:
Hmm.. I had it before just to get that initial call out of the way before getting into the initialize function.

Reply:
`window.speechSynthesis.onvoiceschanged = () => this.initializeVoice();` is setting an event callback, so isn't calling `initializeVoice` directly here.  I think I'm overthinking this; at the very least it all needs to be in curly braces.

Reply:
Yep, I added the curly braces in my last merge!